expression/builtin.go

```diff
@@ -549,6 +549,15 @@ func (b *baseFunctionClass) verifyArgs(args []Expression) error {
 	if l < b.minArgs || (b.maxArgs != -1 && l > b.maxArgs) {
 		return errIncorrectParameterCount.GenByArgs(b.funcName)
 	}
+	for i, arg := range args {
+		if arg == nil {
```

Comment:
fix ut `TestSubstituteCorCol2Constant` in file "expression/util_test.go"

In that situation, builtin function `plus`'s second argument is a `Column` with nil return type, this is invalid in the new expression architecture.